### PR TITLE
added: card component

### PR DESCRIPTION
**Pull request by tusharkandpal**
configured helper & utilties files and added card component

### CODE REVIEW

File: components/cards/card.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  background-color: rgb(255, 241, 254);
+}
+i {
+  padding: 0.2rem 0.4rem;
+}
+i:hover {
+  cursor: pointer;
+  border-radius: 0.3rem;
+  background-color: #e6e3e3;
+}
```
Comment (JantuDeb) [inline components/cards/card.css:75]: Use classes instead of  "<"i> element.

**Review by JantuDeb — APPROVED** [review]
It's looking good.

File: components/cards/card.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  padding: 0.2rem 0.4rem;
+  color: rgb(224, 115, 217);
+}
+.card-link:hover {
+  background-color: rgb(255, 241, 254);
+}
+i {
+  padding: 0.2rem 0.4rem;
+}
+i:hover {
```
Comment (Raghav888) [inline components/cards/card.css:71]: use class name instead of the tag name

File: components/cards/card.css [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+.card-vertical {
+  max-width: 18rem;
+  margin: 0.5rem;
+}
+.card-details-container {
+  display: flex;
+  flex-direction: column;
+}
+.card-details {
+  margin-bottom: 0.5rem;
```
Comment (Raghav888) [inline components/cards/card.css:39]: Try to use var at the padding, margin, radius, etc. That will help you in future projects.

**Review by Raghav888 — APPROVED** [review]
Rest code is good.

File: components/cards/card.css (hunk 1 above)
Comment (tusharkandpal) [reply components/cards/card.css:75]: Yes i will change it @JantuDeb  👍

File: components/cards/card.css (hunk 3 above)
Comment (tusharkandpal) [reply components/cards/card.css:39]: @Raghav888 thanks for the review . im already using helper classes for few cases.. but these values are so random sometimes i dont want to declare every other value as variable..

**Pull request merged**